tests/Functional/FeedManagerTest.php

```diff
@@ -39,7 +39,7 @@ public function testItReturnsFeedCollectionFromValidXml(string $xml, $expectedFe
     }
 
     /**
-     * @dataProvider feedProvider
+     * @dataProvider Linio\SellerCenter\FeedManagerProvider::feedProvider
```

Comment:
Could you import "Linio\SellerCenter\FeedManagerProvider" instaed write in line?

Reply:
Not for dataProvider annotation, unfortunately. At PHPUnit' s doc I didn't find any reference for this but if we use `use statement` the references isn't resolved.